b2g/components/RemoteControlService.jsm

```diff
@@ -325,6 +325,16 @@ this.RemoteControlService = {
     }
   },
 
+  // Refresh the certificate when the time is updated
+  // The getOrCreateCert will validate the certificate itself.
+  // It is invalid when (current time - cert's expired day) < one day,
+  // or the (current time < cert's started day), otherwise we can use
+  onTimeChange: function(aTimestamp) {
+    DEBUG && debug("time is updated to: " + new Date(aTimestamp).toString());
+    this.stop();
+    this.start();
```

Comment:
If the time-change event is not fired when booting up, then the service will be started after the network is connected.